demucs/train.py

```diff
@@ -24,18 +25,40 @@
 from .hdemucs import HDemucs
 from .repitch import RepitchedWrapper
 from .solver import Solver
+from .states import capture_init
 
 logger = logging.getLogger(__name__)
 
 
+class TorchHDemucsWrapper(nn.Module):
+    """Wrapper around torchaudio HDemucs implementation to provide the proper metadata
+    for model evaluation.
+    See https://pytorch.org/audio/main/tutorials/hybrid_demucs_tutorial.html"""
+
+    @capture_init
+    def __init__(self,  **kwargs):
+        super().__init__()
+        try:
+            from torchaudio.prototype.models import HDemucs as TorchHDemucs
```

Comment:
```suggestion
            from torchaudio.models import HDemucs as TorchHDemucs
```

hybrid demucs will be released as a beta feature in our upcoming release (targeting late october) 🙂 

cc @skim0514